cinema_service/settings.py

```diff
@@ -86,8 +91,12 @@
 
 DATABASES = {
```

Comment:
The database settings are currently being loaded from environment variables. This is a good practice for keeping sensitive information out of your code. However, you should provide a default value for each of these keys to prevent KeyError if the environment variable is not set. You can provide a default value like this: os.getenv('POSTGRES_DB', 'default_value').